apps/checkin/views.py

```diff
@@ -1,56 +1,163 @@
+    })
+
+
+@api_view(['GET'])
+def wallet(request):
+
+    headers = request.META
+    if "Authorization" not in headers:
+        return error_response("No authorization header provided.", 401)
+    token = headers["Authorization"].split("Bearer ")[1]
+    group_id = jwt.decode(token, SECRET_KEY)["group_id"]
+    try:
+        group = CheckInGroup.objects.get(id=group_id)
+    except CheckInGroup.DoesNotExist:
+        return error_response("Invalid jwt; no group exists with this id.",
+                              401)
+
+    user = group.applicant
+    cardInfo = EventTicket()
+    cardInfo.addPrimaryField('name', user.name, 'Name')
+    cardInfo.addHeaderField('header', 'October 12-14, 2018',
+                            'Brooklyn Expo Center')
+
+    if user.school:
+        cardInfo.addSecondaryField('loc', user.school, 'School')
+    cardInfo.addSecondaryField('email', user.username, 'Email')
+
+    organizationName = 'TotalityHacks'
+    passTypeIdentifier = 'pass.com.totalityhacks.totalityhacks'
+    teamIdentifier = ''  # TODO: Actually put this here.
+
+    passfile = Pass(cardInfo,
+                    passTypeIdentifier=passTypeIdentifier,
+                    organizationName=organizationName,
+                    teamIdentifier=teamIdentifier)
+
+    passfile.labelColor = 'rgb(255,255,255)'
+    passfile.foregroundColor = 'rgb(255,255,255)'
+
+    # TODO: make sure this is correct
+    passfile.relevantDate = '2018-10-12T10:00-01:00'
+
+    latitude = 40.728157
+    longitude = -73.957797
+
+    location = Location(latitude, longitude)
+    location.distance = 600
+
+    passfile.serialNumber = hashlib.sha256(user.username).hexdigest()
+    passfile.locations = [location]
+    passfile.barcode = Barcode(message=user.username, format=BarcodeFormat.QR)
+
+    dir = os.path.dirname(__file__)
+
+    passfile.addFile('icon.png',
+                     open(os.path.join(dir, 'passbook/icon.png'), 'r'))
+    passfile.addFile('icon@2x.png',
+                     open(os.path.join(dir, 'passbook/icon@2x.png'), 'r'))
+    passfile.addFile('icon@3x.png',
+                     open(os.path.join(dir, 'passbook/icon@3x.png'), 'r'))
+
+    passfile.addFile('logo.png',
+                     open(os.path.join(dir, 'passbook/logo.png'), 'r'))
+    passfile.addFile('logo@2x.png',
+                     open(os.path.join(dir, 'passbook/logo@2x.png'), 'r'))
+    passfile.addFile('logo@3x.png',
+                     open(os.path.join(dir, 'passbook/logo@3x.png'), 'r'))
+
+    passfile.addFile('background.png',
+                     open(os.path.join(dir, 'passbook/background.png'), 'r'))
+    passfile.addFile('background@2x.png',
+                     open(os.path.join(dir, 'passbook/background@2x.png'),
+                          'r'))
+    passfile.addFile('background@3x.png',
+                     open(os.path.join(dir, 'passbook/background@3x.png'),
+                          'r'))
+
+    key_path = 'secure/'  # TODO: Make sure we add the key here
```

Comment:
task! I'll probably need to help you make these certs

Reply:
Yes. Definitely you'll need to help with that.